### DISCUSSION TITLE
[BUG] Consulta sobre el formato de las horas de modificación

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#168] [4. Bug Reports] [answered]
Hola quisiera saber sobre el formato de la hora. Me di cuenta cuando creé una task (con asset unico), la fecha de modificación está en formato 12 o es -12?adjunto un ss

![image](https://user-images.githubusercontent.com/61419471/117732825-86e52980-b1be-11eb-9d11-b1ad663b6088.png)

**Comment by rafaelwalter** [comment]
@gmoplaza cual es el formato de display de fecha en las tablas?

**Comment by gmoplaza** [comment, marked as answer]
@NicolasPLG a que te refieres con 12 o -12? no sé que significa -12.
Si la pregunta es de que forma se muestran las horas, por la imagen adjunta puedo ver que es 01-12.
En caso de querer que sean 00-23, puedes levantar el requerimiento con quien corresponda y podemos cambiar todas las horas del admin.

**Reply by acedward** [reply]
@gmoplaza igual si usamos hh:mm en vez de HH:mm deberiamos incluir el AM/PM

**Reply by NicolasPLG** [reply]
@gmoplaza el -12, me faltó agregar el gtm, porque si muestra solo 12, no sabes si es am/pm y como bien dice @acedward se debería incluir am/pm para diferenciar el tiempo y eso creo que debería estar a nivel de producto.
=>

**Reply by acedward** [reply]
Las fechas deberian siempre estar en tu hora local, si no dice lo contrario.
Si no es así por favor dejanos saber.

**Reply by NicolasPLG** [reply]
Eso lo entiendo, pero qué pasaría si en el caso de una auditoría de datos, revisan los registros y sale las 4:00 que hubo una modificación, no sabrás si fue a las 4 am o 4 pm, por allí iba m consulta. En el caso que muestre solo la hora desde 1 a 12, existe la posibilidad de agregar el AM/PM para entender las horas?. Se que no tiene gran impacto y puede quedar en el backlog infinito, pero creo que debería poder diferenciar la hora de la mañana con la hora de la tarde.

**Reply by gmoplaza** [reply]
@acedward pasemos todo a HH:mm, me debería tomar 30min a una hora hacerlo

**Reply by gmoplaza** [reply]
@NicolasPLG cree la solicitud #2871 para arreglar el problema.

**Reply by NicolasPLG** [reply]
Gracias @gmoplaza por la respuesta. :>
